lib/node_modules/@stdlib/math/base/complex/divide/lib/index.js

```diff
@@ -10,6 +10,15 @@
 *
 * var v = cdiv( -13.0, -1.0, -2.0, 1.0 );
 * // returns [ 5.0, 3.0 ]
+*
+* @example
+* var out = new Array( 2 );
```

Comment:
Ah, for module JSDoc definitions, each example should be independently runnable. Hence, for the second example,

```
* @example
* var cdiv = require( '@stdlib/math/base/complex/divide' );
*
* var out = new Array( 2 );
* var v = cdiv( out, -13.0, -1.0, -2.0, 1.0 );
* // returns [ 5.0, 3.0 ]
*
* var bool = ( v === out );
* // returns true
*/
```